### PR TITLE
added a wrapper around the `entt::entity` type for easier usage, added demonstration code

### PR DESCRIPTION
**Pull request by mgerhold**

### CODE REVIEW

File: src/silly_sprites/entity.hpp [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        [[nodiscard]] std::tuple<Components&...> get_components() {
+            return m_registry->get<Components...>(m_identifier);
+        }
+
+        template<typename... Components>
+        [[nodiscard]] std::tuple<Components const&...> get_components() const {
+            return m_registry->get<Components...>(m_identifier);
+        }
+
+        friend std::ostream& operator<<(std::ostream& ostream, const Entity& entity) {
```
Comment (PurpurTentakel97) [inline src/silly_sprites/entity.hpp:43]: wrong const

**Review by PurpurTentakel97 — CHANGES_REQUESTED** [review]

File: src/silly_sprites/entity.hpp [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "utils.hpp"
+#include <entt/entt.hpp>
+#include <iostream>
+
+namespace sly {
+
+    class Entity final {
+    public:
```
[the remark this answers is not in the document]
Comment (mgerhold) [reply src/silly_sprites/entity.hpp:10]: I opened a [discussion](https://github.com/mgerhold/silly-sprites/discussions/43) on that topic. Let's talk about it there :)

File: src/silly_sprites/native_script.hpp [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#pragma once
+
```
[the remark this answers is not in the document]
Comment (mgerhold) [reply src/silly_sprites/native_script.hpp:2]: Do you mean between `#pragma once` and the `#include`s? I prefer to put a newline there. Clang-format does not force this, though. Should we add this to the styleguide? Would you put the newline there or not?

[the remark this answers is not in the document]
Comment (PurpurTentakel97) [reply src/silly_sprites/native_script.hpp:2]: i think i like it.
so lets add it to the styleguide

**Review by PurpurTentakel97 — APPROVED** [review]

**Pull request merged**